### DISCUSSION TITLE
I have a problem about install bonsai in my new computer

### DISCUSSION THREAD

**Discussion by kairos-bar** [bonsai-rx/bonsai#1496] [Q&A]
I'm a colleage student and I have been using Bonsai steadily and well.

And, I want to install bonsai in my new computer. 
But, My problem is that the above phenomenon occurs.

How to I solve it? Please give us a lot of help.

![20230727_143619](https://github.com/bonsai-rx/bonsai/assets/137870547/e716f410-2d8e-447a-9828-5b094d01f689)

**Comment by bruno-f-cruz** [comment]
The MyGet server is down. Wait for a few hours it will hopefully come back!
